pkg/aggregate/aggregate.go

```diff
@@ -194,7 +194,10 @@ type AggregatesList []Aggregate
 
 // Append a value to all aggregates
 func (a AggregatesList) Aggregate(t int64, val interface{}) {
-	v := val.(float64)
+	v, ok := val.(float64)
```

Comment:
If you want to skip the `variants` you should call aggregate at all

Reply:
It happens automatically when ingesting data. Now if you have one tsdb with 2 metrics one of them is normal (like cpu) the other is of variant type (like log).
you created the table with default aggregates that are being aggregated on ingest for every metric in the DB so we need to ignore non-floats.
At least until a better solution.